### PR TITLE
✨ [PRTL-1452] upload case set

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: src/packages/@ncigdc/components/Modals/UploadSet/MappingTable/CaseMappingTable.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    const matchedCases = Object.entries(
+      matched.reduce((acc, hit) => {
+        const caseId = caseMap[hit].submitter_id.toUpperCase();
+        acc[caseId] = (acc[caseId] || []).concat(hit);
+        return acc;
+      }, {}),
+    )
+      .map(([caseId, submitted]) => {
+        const caseData = caseMap[caseId];
+        var temp = CASE_ID_FIELDS.map((idField, i) => {
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Modals/UploadSet/MappingTable/CaseMappingTable.js]: I don't see this getting reassigned / used for hoisting?

File: src/packages/@ncigdc/utils/validateIds.js [hunk 2]
```diff
@@ -1,30 +1,70 @@
-import { get } from 'lodash';
-
 import { fetchApiChunked } from '@ncigdc/utils/ajax';
 
+function _getIds(obj, path) {
+  const segment = path.shift();
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/utils/validateIds.js]: [segment] = path ? kinda weird to see shift all of a sudden

File: src/packages/@ncigdc/components/Modals/UploadSet/MappingTable/CaseMappingTable.js (hunk 1 above)
Comment (Jephuff) [inline src/packages/@ncigdc/components/Modals/UploadSet/MappingTable/CaseMappingTable.js]: ahh yeah, that was a leftover from some debugging.

File: src/packages/@ncigdc/utils/validateIds.js (hunk 2 above)
Comment (Jephuff) [inline src/packages/@ncigdc/utils/validateIds.js]: I was using it to also modify path, but I can refactor to not use shift 👍

**Pull request merged**